source/features/use-first-commit-message-for-new-prs.tsx

```diff
@@ -0,0 +1,38 @@
+import select from 'select-dom';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+import * as textFieldEdit from 'text-field-edit';
+
+import features from '.';
+import {looseParseInt} from '../github-helpers';
```

Comment:
Can you move this out of github-helpers in the other PR? It's a generic helper

Reply:
Done